gast_to_code/gast_to_code_main.py

```diff
@@ -111,6 +111,29 @@ def gast_to_js_if(gast):
 
     return out
 
+def gast_to_js_func_declarations(gast):
+    name = gast_to_code(gast["id"], "js")
+    args = gast_to_code(gast["params"], "js")
+    body = list_helper(gast["body"], "js", "\n\t")
+    out = "function " + name
+    out += "(" + args + ") {\n\t"
```

Comment:
This looks like it would only indent the first line of the body. I think formatting should be delegated to some other system to let this focus on generating correct code.

Reply:
Currently we only have support for one level of indent. The list_helper called in body separates each body statement with \n\t so every line is indented. We're added nested indents to the taiga and will address later.